storybook/components/CustomCanvas/CustomCanvas.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react"
+import { Source, SourceProps, Story, Unstyled } from "@storybook/blocks"
+import classnames from "classnames"
+import { v4 } from "uuid"
+import styles from "./CustomCanvas.module.scss"
+
+export type CustomCanvasProps = {
+  of: SourceProps["of"]
+  initialIsOpen?: boolean
+  className?: string
+}
+
+/**
+ * An almost-duplicate of Storybook's <Canvas>.
+ * Only use this when Canvas prevents your component rendering correctly (eg. using a popover).
```

Comment:
Does it present a problem in some cases? Why not use this for all components? I'd rather we use one for all so we avoid mixed communication.

Reply:
It doesn't include the toolbar with the zoom options etc 😬 Given that we don't allow people to see individual Stories nowadays through the site, I figured it's best to leave them using the base?

Reply:
I don't know how important zoom is though, I'd be happy to make the bet that no one uses it 😄

Reply:
We saw Jason using it the other day in a call 😂

Reply:
Haha then maybe we call this the `NoClipCanvas` or something, specifically to resolve the popover issue and help someone decide when to use it.

Reply:
Smart! Let's go with that :D